src/components/Data/Collections/Watch.vue

```diff
@@ -370,6 +365,12 @@
         }
 
         this.notifications.push(this.notificationToMessage(result))
+
+        // Auto scroll
+        const div = document.getElementById('notification-container')
```

Comment:
I used it on DOM element of the current component, you can't access a ref from a child element